docs/operate/reference/viam-micro-server/micro-troubleshooting.md

```diff
@@ -13,16 +13,9 @@ aliases:
 # updated: ""  # When the content was last entirely checked
 ---
 
-## Micro-RDK CLI
-
-The Viam Micro-RDK installer is a command line tool that you can also use to troubleshoot.
-It allows you to:
-
-- View logs locally and debug issues when your device has not connected to the Viam app
-- Change wifi credentials on your microcontroller
-- Overwrite firmware using a serial cable
-
-For more information on the `micro-rdk-installer` CLI usage options, see the [Micro-RDK Installer GitHub README](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
+{{% alert title="Tip" color="tip" %}}
+The [Viam Micro-RDK installer](/operate/reference/viam-micro-server/manage-micro/#micro-rdk-installer) is a command line tool that can be helpful for troubleshooting.
+{{% /alert %}}
 
 ## Common issues
```

Comment:
Can we move this and the other troubleshooting tips that are currently in "Set up an ESP32" together onto a "Troubleshooting" sub-page under "Set up an ESP32"?

That way all content about MicroRDK will be in two places: "Set up an ESP32" and "ESP32 modules".
It's a little easier for me to remember (and maybe for users to find).

Reply:
There will still be the parent page of this one in reference, but that one's not about usage, so sure! I may create a troubleshooting section for flashing and a troubleshooting section for other stuff though...TBD